packages/delivery-node/request.js

```diff
@@ -0,0 +1,43 @@
+const http = require('http')
+const https = require('https')
+const { parse } = require('url')
+
+module.exports = ({ url, headers, body, agent }, cb) => {
+  let didError = false
+  const onError = (err) => {
+    if (didError) return
+    didError = true
+    cb(err)
+  }
+
+  const parsedUrl = parse(url)
+  const secure = parsedUrl.protocol === 'https:'
+  const transport = secure ? https : http
+  const req = transport.request({
+    method: 'POST',
+    hostname: parsedUrl.hostname,
+    port: parsedUrl.port,
+    path: parsedUrl.path,
+    headers
+  })
+  req.on('error', onError)
+  req.on('response', res => {
+    bufferResponse(res, (err, body) => {
+      if (err) return onError(err)
+      if (res.statusCode !== 200) {
```

Comment:
The session api returns 202 on success rather than 200 - https://bugsnagsessiontrackingapi.docs.apiary.io/#reference/0/session/report-a-session-starting

Reply:
we should check if its 2xx